frontend/app/quest/view3.html

```diff
@@ -0,0 +1,69 @@
+<html >
+    <head>
+        <meta charset="utf-8">
+        <meta name="description" content="Hello">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="shortcut icon" href="https://s3-us-west-2.amazonaws.com/itreeware/PL_Favicon.png" type="image/x-icon">
+        <link rel="icon" href="https://s3-us-west-2.amazonaws.com/itreeware/PL_Favicon.png" type="image/x-icon">
+        <title>PlanetLAB> Quest Builder</title>
+        <link rel="stylesheet" href="https://s3-us-west-2.amazonaws.com/itreeware/planetLabStyle.css" type="type/css">
+        </head>
+
+<img  src="https://s3-us-west-2.amazonaws.com/itreeware/PL--Header.png" alt="PlanetLAB |Freedom Games"   >
+        <ol class="controlNav"   >
+            <li class="questControlNav"  >
+                <a  href="Quest_Basic.html"
+                   >Basic Info</a>
+            </li>
+            <li class="questControlNav"> 
+                <a  href="./Quest_Activity.html">Activity Details</a>
+            </li>
+            <li class="questControlNav"> 
+                <a  href="./Quest_for_teachers.html">For Teachers</a>
+            </li>
+            <li class="questControlNav"> 
+                <a  href="./Quest_Review.html">Review</a>
+            </li>
+        </ol>
+        <h1>Quest Builder Form </h1>
+        <div style="padding=5%;">
```

Comment:
This isn't valid CSS, you probably meant padding: 5%; but this should be in an external CSS file anyways. If you're having trouble targeting this div, we can add a class or something -- if this was a quick fix I understand, but let's take it out